src/Resources/assets/shop/js/wishlistModal.js

```diff
@@ -82,7 +82,7 @@ export class WishlistModal {
         });
 
         confirmBtn.addEventListener('click', () => {
-            if(input && input.value === '') {
+            if(input && (input.value === '' || input.value.match(/ /gi).length === input.value.length)) {
```

Comment:
More optimal solution would be
`return str === null || str.match(/^ *$/) !== null;`
Also what about tabs instead of spaces whitespace?